experimental/packages/otlp-grpc-exporter-base/src/configuration/otlp-grpc-env-configuration.ts

```diff
@@ -152,8 +153,8 @@ function readFileFromEnv(
   nonSignalSpecificEnvVar: string,
   warningMessage: string
 ): Buffer | undefined {
-  const signalSpecificPath = process.env[signalSpecificEnvVar]?.trim();
-  const nonSignalSpecificPath = process.env[nonSignalSpecificEnvVar]?.trim();
+  const signalSpecificPath = getStringFromEnv(signalSpecificEnvVar);
+  const nonSignalSpecificPath = getStringFromEnv(nonSignalSpecificEnvVar);
 
   const filePath = fallbackIfNullishOrBlank(
```

Comment:
I think this could be simplified to (untested):

```ts
const filePath = getStringFromEnv(signalSpecificEnvVar) ?? getStringFromEnv(nonSignalSpecificEnvVar);
```

Then, with this suggestion and those above, `fallbackIfNullishOrBlank` is now unused in this file and can be removed.